service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/StandardDirectionOrderIssuedEmailContentProvider.java

```diff
@@ -35,4 +53,56 @@ public AllocatedJudgeTemplateForSDO buildNotificationParametersForAllocatedJudge
 
         return allocatedJudgeTemplate;
     }
+
+    public CTSCTemplateForSDO buildNotificationParametersForCTSC(CaseDetails caseDetails) {
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        HearingBooking hearing = caseData.getFirstHearing().orElseThrow(NoHearingBookingException::new);
+
+        CTSCTemplateForSDO ctscTemplateForSDO = new CTSCTemplateForSDO();
+        ctscTemplateForSDO.setRespondentLastName(getFirstRespondentLastName(caseData.getRespondents1()));
+        ctscTemplateForSDO.setCallout(buildCallout(caseData));
+        ctscTemplateForSDO.setCourtName(getCourtName(caseData.getCaseLocalAuthority()));
+        ctscTemplateForSDO.setHearingNeedsPresent(getHearingNeedsPresent(hearing));
+        ctscTemplateForSDO.setHearingNeeds(buildHearingNeedsList(hearing.getHearingNeedsBooked()));
+        ctscTemplateForSDO.setHearingNeedsDetails(hearing.getHearingNeedsDetails());
+        ctscTemplateForSDO.setCaseUrl(getCaseUrl(caseDetails.getId()));
+        ctscTemplateForSDO.setDocumentLink(concatUrlAndMostRecentUploadedDocumentPath(xuiBaseUrl,
+            caseData.getStandardDirectionOrder().getOrderDoc().getBinaryUrl()));
+
+        return ctscTemplateForSDO;
+    }
+
+    private String buildCallout(final CaseData caseData) {
+        return "^" + buildSubjectLineWithHearingBookingDateSuffix(caseData.getFamilyManCaseNumber(),
+            caseData.getRespondents1(),
+            caseData.getFirstHearing().orElse(null));
+    }
+
+    private String getCourtName(String courtName) {
+        return hmctsCourtLookupConfiguration.getCourt(courtName).getName();
+    }
+
+    private List<String> buildHearingNeedsList(List<HearingNeedsBooked> hearingNeedsBooked) {
+        List<String> list = new ArrayList<>();
+
+        if (hearingNeedsBooked == null || hearingNeedsBooked.isEmpty()) {
+            return list;
+        }
+
+        for (HearingNeedsBooked hearingNeed : hearingNeedsBooked) {
+            if (hearingNeed == NONE) {
+                return emptyList();
+            }
+            if (hearingNeed != SOMETHING_ELSE) {
+                list.add(hearingNeed.getLabel());
+            }
+        }
+        return list;
+    }
+
+    private String getHearingNeedsPresent(HearingBooking hearingBooking) {
+        return hearingBooking.getHearingNeedsBooked() == null
+            || hearingBooking.getHearingNeedsBooked().contains(NONE) ? "No" : "Yes";
+    }
```

Comment:
Again might make for a nice method on HearingNeedsBooked

Reply:
I think this relates more directly to the content provider as we are returning "No" or "Yes" for the gov notify parameter, so I'd say keep it here